app/models/header_lookup.rb

```diff
@@ -1,14 +1,14 @@
 class HeaderLookup
-  MEMOIZED_MESH_FILE = "memoized_mesh.txt"
-  MEMOIZED_LCSH_FILE = "memoized_lcsh.txt"
-  MEMOIZED_LCNAF_FILE = "memoized_lcnaf.txt"
+  MEMOIZED_MESH_FILE = "memoized_mesh.json"
+  MEMOIZED_LCSH_FILE = "memoized_lcsh.json"
+  MEMOIZED_LCNAF_FILE = "memoized_lcnaf.json"
 
   SEARCHABLE_MESH_FILE = "subjects_mesh.jsonl"
   SEARCHABLE_LCSH_FILE = "subjects_lcsh.jsonl"
   # Needs to be in config because file is too large for VCS
   SEARCHABLE_LCNAF_FILE = ENV["SUBJECTS_LCNAF"]
 
-  ABSENT_SUBJECT = :absent
+  ABSENT_SUBJECT = "absent"
```

Comment:
In order for the JSON library to parse correctly we need to use the string `"absent"` and not the symbol `:absent`. It's pretty annoying , but 🤷